src/app/components/client/Chart.tsx

```diff
@@ -153,7 +153,12 @@ export const DoughnutChart = (props: Props) => {
   );
 
   const getPromptContent = () => {
-    if (!props.scanInProgress && props.isEligibleForPremium) {
+    if (
+      !props.scanInProgress &&
+      props.isEligibleForPremium &&
+      (props.totalNumberOfPerformedScans === undefined ||
+        props.totalNumberOfPerformedScans === 0)
+    ) {
```

Comment:
It would be best to add a unit test here to cover this code branch and make it easier to catch regressions.